README.md

```diff
@@ -2,3 +2,62 @@ Msgstat
 ----------
 Msgstat provides a CLI tool which analyses status-go log files and generates corresponding output of information 
 related to messages delivered over whisper (either rpc or p2p).
+
+## Commands
+
+### `-readfile`
+
+Msgstat `-readfile` expects to be provided a path to the proper log file which will be processed to generate reports.
```

Comment:
`readfile` is a command, not an option. Should it be prefixed with `-`?

Reply:
True.